comp-sci/networking/metrics/what-is-packet-loss.md

```diff
@@ -47,7 +47,7 @@ The causes of packet loss vary, yet the most common ones are:
 ---
 ## Revision
 
-_Packet loss_ is measured
+#### *Packet loss* is measured
```

Comment:
Our format needs a third-level header here to specify the "headline", or it will throw an error / not have a headline

Reply:
Absolutely, I'll get that taken care of! I look forward to continuing to contribute!